Updated cttso to pieriandx pipeline to consider research samples and split panel from sample type

alexiswl opened the pull request:
## Major update

Split out panel type from sample type, added glims_is_research and panel_type columns to cttso lims

See https://github.com/umccr/cttso-ica-to-pieriandx/pull/19 or Changelog.md for more info.  


## Additional updates
* Dependabot updates for json5 #17
* Updated readme to run payloads manually #10
* Fixed windows line endings on some files #18 
* Added in deployment diagram for explanation of how panel and sample types are chosen for PierianDx based off RedCap and GLIMS information #20

## Issues fixed
* #14 
* #15

mhlunimelb commented:
Looks great to me Alexis.
I tested the dev branch with L2201567 ( a library the curators want multiple copies for)  it all worked fine.
The only things I wasn't sure about with the" Running payloads manually (step-by-step) tutorial" was whether 
"cd deploy/cttso-ica-to-pieriandx" should be "cd deploy/cttso-ica-to-pieriandx-cdk" ?

I also thought maybe it would help to clarify the the required ica_workflow_run_id is the "TSO_CTDNA_TUMOR_ONLY" workflow id and it can also be found by hunting for the library id in biobots?
Thanks, Helen

alexiswl commented:
> was whether "cd deploy/cttso-ica-to-pieriandx" should be "cd deploy/cttso-ica-to-pieriandx-cdk" ?

Yep well spotted, updated in dev branch (35b7b64c14745369daa8fff75b7a9addf043c565)

> I also thought maybe it would help to clarify the the required ica_workflow_run_id is the "TSO_CTDNA_TUMOR_ONLY" workflow id and it can also be found by hunting for the library id in biobots?

Have also added something in (35b7b64c14745369daa8fff75b7a9addf043c565)
